Kiosk/Sale Artwork Details/SaleArtworkDetailsViewController.swift

```diff
@@ -199,7 +199,7 @@ class SaleArtworkDetailsViewController: UIViewController {
             .rx.observe(NSNumber.self, "highestBidCents")
             .map { observeredCents -> Bool in
                 guard let cents = observeredCents else { return false }
-                return (cents as Int) > 0
+                return (cents as! Int) > 0
```

Comment:
pedantic: perhaps this would look/feel better with the typecast in the guard as `guard let cents = observeredCents as Int else { return false }`